src/components/Select/components/Control/Control.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {CnMods, block} from '../../../utils/cn';
+import {Icon} from '../../../Icon';
+import {Chevron} from '../../../icons/Chevron';
+import {SelectProps} from '../../types';
+
+import './Control.scss';
+
+const b = block('select-control');
+
+type ControlProps = {
+    view: NonNullable<SelectProps['view']>;
+    size: NonNullable<SelectProps['size']>;
+    pin: NonNullable<SelectProps['pin']>;
+    value: string[];
+    width?: SelectProps['controlWidth'];
+    label?: string;
+    placeholder?: SelectProps['placeholder'];
+};
+
+export const Control = (props: ControlProps) => {
+    const {view, size, pin, value, width, label, placeholder} = props;
+    const showValues = Boolean(value.length);
+    const showPlaceholder = Boolean(placeholder && !showValues);
+    const controlMods: CnMods = {
+        view,
+        size,
+        pin,
+        ...(typeof width === 'string' && {width}),
+    };
+    const controlInlineStyles: React.CSSProperties = {};
+
+    if (typeof width === 'number') {
+        controlInlineStyles.width = width;
+    }
+
+    return (
+        <div className={b(controlMods)} style={controlInlineStyles} tabIndex={1}>
```

Comment:
Are we planning to make it accessibility friendly?
Why use `div` instead of `button`?
What about `role="combobox"`, `aria-expanded="..."`, `aria-haspopup="listbox"`, e.t.c. ?

references:
-  https://w3c.github.io/aria/#combobox
- https://w3c.github.io/aria-practices/examples/combobox/combobox-select-only.html

Reply:
Change wrapper to `button`, add `role` and `aria-haspopup`